src/modm/platform/timer/stm32/advanced.hpp.in

```diff
@@ -338,6 +338,24 @@ public:
 		TIM{{ id }}->BDTR = flags;
 	}
 
+	/**
+	 * sets the update rate of the compare registers
+	 * @details in PWM mode uploadRate means:
```

Comment:
hmm... im pretty sure the description is not totally correct:
RCR -> RepetitionCountRegister
so basically for that number of times it will reload the counter and than stop and generate an interrupt

Reply:
Maybe just calling the setter: setRepetitionCount() is better?

Reply:
This was the description on STM's reference manual, but it is pretty much a cryptic one. I hope this approach is better now.